spec/system/admin/subscriptions_spec.rb

```diff
@@ -434,6 +434,15 @@ def open_subscription_products_panel
                     expect(subscription.bill_address.firstname).to eq 'John'
                     expect(subscription.ship_address.firstname).to eq 'John'
                   end
+
+                  it 'creates standing line items' do
+                    subscription = Subscription.last
+                    subscription_line_item = subscription.subscription_line_items.first
+
+                    expect(subscription.subscription_line_items.count).to eq 1
+                    expect(subscription_line_item.variant).to eq shop_variant
+                    expect(subscription_line_item.quantity).to eq 3
+                  end
```

Comment:
I'm all for testing smaller units, but because these last two examples require no interaction, I would suggest that all the expectations are moved into the one example (that is, `it 'has selector for price, input for quantity and selector for total price' do`). 
This will save having to rebuild the exact same browser state two extra times, saving time.